leap/setup.cfg

```diff
@@ -0,0 +1,11 @@
+[bumpversion]
+current_version = 0.1
+tag = True
+commit = True
+
+[bumpversion:file:leap/__version__.py]
+
+[coverage:run]
```

Comment:
The coverage settings can be removed since there won't be CI tests running on this